src/main/flight/pid.c

```diff
@@ -344,6 +344,12 @@ static void pidApplyRateController(const pidProfile_t *pidProfile, pidState_t *p
         pidState->errorGyroIfLimit = ABS(pidState->errorGyroIf);
     }
 
+#ifdef USE_SERVOS
```

Comment:
Ok, I see. Yes, we still support CJMCU (as well as CC3D and NAZE) - it's a good check against code bloat.